frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -10,34 +17,57 @@ const StyledInputField = styled(InputField)(
   spacing.withWhiteSpace({ marginBottom: 6 })
 );
 
-export const SearchForm = ({ indicator }: { indicator: string }) => {
-  const initialState: SearchFormState = {
-    indicator: '',
-    message: null,
-    errors: {},
-  };
-  const [_state, formAction] = useActionState(searchIndicator, initialState);
+export const SearchForm = ({
+  searchFormState,
+}: {
+  searchFormState: SearchFormState;
+}) => {
+  const [state, formAction] = useActionState(searchIndicator, searchFormState);
 
   return (
-    <>
-      <H1>Search</H1>
+    <main>
       <form action={formAction}>
+        {state.message && (
+          <ErrorSummary
```

Comment:
The [Gov.UK frontend guidance for the Error Summary component](https://design-system.service.gov.uk/components/error-summary/) says:
> When a user makes an error, you must show both an error summary and an [Error message component](https://design-system.service.gov.uk/components/error-message/) next to each answer that contains an error.

So I think we need to do that too.

Reply:
Discussed this with Fawad, he logic for not doing this is that when the page has two fields leaving either of them blank is not an error, it's only a problem when both are blank. In this case you can't really say which of the fields is in error so I don't think it makes sense to highlight one or the other.